app/api/(auth)/connect/rsa/route.ts

```diff
@@ -3,13 +3,9 @@ import { withBasePath } from "@/app/_lib/util/links"
 import { cookies } from "next/headers"
 import { NextRequest, NextResponse } from "next/server"
 
-// const RSA_BACKEND_URL = process.env.RSA_BACKEND_URL // 'https://dev.radarbasedev.co.uk/rest-sources/backend'
-// const RSA_FRONTEND_URL = 'https://dev.radarbasedev.co.uk/rest-sources/authorizer'
-// const RSA_REDIRECT_URL = 'https://dev.radarbasedev.co.uk/kratos-ui/'
+const RSA_BACKEND_URL = process.env.RSA_BACKEND_URL
+const RSA_FRONTEND_URL = process.env.RSA_FRONTEND_URL
```

Comment:
@callumstew I've deployed the `dev` image, but I think we only have the backend env. I will add `RSA_FRONTEND_URL`.